### PR TITLE
#500 Default position coordinates are not integers

### PR DESCRIPTION
**Pull request by langonginc**

### CODE REVIEW

**Comment by thekingofcity** [comment]
Or maybe we can add `roundToNearestN` at
https://github.com/railmapgen/rmp/blob/0a7b0f2fc4a19414ac889e4c1d3829a7173cdabe/src/components/svg-wrapper.tsx#L103

**Comment by langonginc** [comment]
> Or maybe we can add `roundToNearestN` at
> 
> https://github.com/railmapgen/rmp/blob/0a7b0f2fc4a19414ac889e4c1d3829a7173cdabe/src/components/svg-wrapper.tsx#L103
> 
> 

I don't think this is what I have edited.

**Comment by thekingofcity** [comment]
> > Or maybe we can add `roundToNearestN` at
> > https://github.com/railmapgen/rmp/blob/0a7b0f2fc4a19414ac889e4c1d3829a7173cdabe/src/components/svg-wrapper.tsx#L103
> 
> I don't think this is what I have edited.

Yeah, but we can save a new function with this?

**Review by thekingofcity — APPROVED** [review]
LGTM

File: src/components/svg-wrapper.tsx [hunk 1]
```diff
@@ -100,8 +100,8 @@ const SvgWrapper = () => {
             graph.current.addNode(`stn_${rand}`, {
                 visible: true,
                 zIndex: 0,
-                x: svgX,
-                y: svgY,
+                x: roundToNearestN(svgX, 5),
+                y: roundToNearestN(svgY, 5),
```
Comment (langonginc) [inline src/components/svg-wrapper.tsx:104]: Is that ok?

Comment (langonginc) [reply src/components/svg-wrapper.tsx:104]: Oh, I see.

**Comment by thekingofcity** [comment]
Maybe we can merge this without testing? This change should be simple and straightforward :)

**Comment by langonginc** [comment]
> Maybe we can merge this without testing? This change should be simple and straightforward :)
> 
> 

Yes, same with me.

**Pull request merged**